src/Tribe/Customizer/Global_Elements.php

```diff
@@ -121,59 +172,156 @@ public function get_css_template( $template ) {
 				}
 			';
 
+			// overrides for tec components/full/_events-bar.pcss
 			$template .= '
-				@keyframes tribe-common-c-loader-bounce {
-					0% {}
-					50% { background-color: <%= global_elements.accent_color %>; }
-					100% {}
+				.tribe-events .tribe-events-c-events-bar__search-button:before {
+					background-color: <%= global_elements.accent_color %>;
+				}
+			';
+
+			// overrides for tec components/full/_ical-link.pcss
+			$template .= '
+				.tribe-events .tribe-events-c-ical__link {
+					border-color: <%= global_elements.accent_color %>;
+					color: <%= global_elements.accent_color %>;
+				}
+			';
+
+			$template .= '
+				.tribe-events .tribe-events-c-ical__link:hover,
+				.tribe-events .tribe-events-c-ical__link:focus,
+				.tribe-events .tribe-events-c-ical__link:active {
+					background-color: <%= global_elements.accent_color %>;
+				}
+			';
+
+			// overrides for tec components/full/_view-selector.pcss
+			$template .= '
+				.tribe-events .tribe-events-c-view-selector__button:before {
+					background-color: <%= global_elements.accent_color %>;
+				}
+			';
+
+			// overrides for tec views/full/list/_event.pcss
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: f2bd16b607bb3fa2da253a2e9349d4c3</sub>